alphagov/paas-admin
# Update Region tag focus state

## What
Update the region tag link to have proper focus state styles

This work
- removes existing, duplicate styles of the [Tag component](https://design-system.service.gov.uk/components/tag/)

- updates the component className to add the `govuk-tag` class for base styles and then the `app-region-tag` class for app specific overrides and `--<region>` modifier class, in line with [BEM](https://css-tricks.com/bem-101/) methodology

## How to review

- build locally 
- use keyboard to tab to the IRELAND and LONDON tag and check focus styles are black text on yellow background (same as link focus styles)

## Visual changes

### Mobile (no change)
<img width="344" alt="Screenshot 2020-03-10 at 09 38 27" src="https://user-images.githubusercontent.com/3758555/76300233-4f29db80-62b4-11ea-969d-7b220c99aeaf.png">

### Desktop (focus)

**Before**
<img width="111" alt="Screenshot 2020-03-10 at 09 33 40" src="https://user-images.githubusercontent.com/3758555/76300308-6668c900-62b4-11ea-8ede-2f9efe34bfc1.png">
<img width="116" alt="Screenshot 2020-03-10 at 09 35 50" src="https://user-images.githubusercontent.com/3758555/76300310-6668c900-62b4-11ea-93f7-78a828141cfc.png">

**After**
<img width="121" alt="Screenshot 2020-03-10 at 09 33 22" src="https://user-images.githubusercontent.com/3758555/76300330-71235e00-62b4-11ea-972a-b1bf7fe04c72.png">
<img width="110" alt="Screenshot 2020-03-10 at 09 35 30" src="https://user-images.githubusercontent.com/3758555/76300332-71bbf480-62b4-11ea-818a-617831e356e8.png">
